gunicorn/workers/gtornado.py

```diff
@@ -81,9 +88,13 @@ def run(self):
         self.ioloop = IOLoop.instance()
         self.alive = True
         self.server_alive = False
-        if IOLOOP_PARAMETER_REMOVED:
-            PeriodicCallback(self.watchdog, 1000).start()
-            PeriodicCallback(self.heartbeat, 1000).start()
+        self.callbacks = []
```

Comment:
Style nit: Perhaps we could move `self.callbacks = []` into the `if TORNADO5:` branch? Other than just one minor comment, this looks good to me. Thank you!

Reply:
Agreed, it's cleaner. Fixed! Thank you for the guide :)